lib/Dog/dogs_list_page.dart

```diff
@@ -51,9 +53,19 @@ class _DogsListPageState extends State<DogsListPage> {
         backgroundColor: Colors.orangeAccent,
       ),
       floatingActionButton: FloatingActionButton(
+        heroTag: "btn1",
```

Comment:
do czego to służy? xd

Reply:
Bez tego się wywalało, bo nie może być dwóch lub więcej `FloatingActionButton`ów bez taga w jakichś tam określonych przypadkach i akurat to był ten przypadek